db/db_impl/db_impl_secondary.cc

```diff
@@ -956,23 +956,10 @@ Status DB::OpenAndCompact(
   compaction_input.db_options.file_checksum_gen_factory =
       override_options.file_checksum_gen_factory;
   compaction_input.db_options.statistics = override_options.statistics;
-  compaction_input.column_family.options.comparator =
-      override_options.comparator;
-  compaction_input.column_family.options.merge_operator =
-      override_options.merge_operator;
-  compaction_input.column_family.options.compaction_filter =
-      override_options.compaction_filter;
-  compaction_input.column_family.options.compaction_filter_factory =
-      override_options.compaction_filter_factory;
-  compaction_input.column_family.options.prefix_extractor =
-      override_options.prefix_extractor;
-  compaction_input.column_family.options.table_factory =
-      override_options.table_factory;
-  compaction_input.column_family.options.sst_partitioner_factory =
-      override_options.sst_partitioner_factory;
+  compaction_input.db_options.listeners = override_options.listeners;
+
   compaction_input.column_family.options.table_properties_collector_factories =
       override_options.table_properties_collector_factories;
-  compaction_input.db_options.listeners = override_options.listeners;
```

Comment:
Is this related? The `listeners` field is still defined in `CompactionServiceOptionsOverride`.

Reply:
It was moved to line 959

Reply:
Sorry, still not fully awake this morning